How to exclude files from download tracking (#379)
piwik/developer-documentation: docs/4.x/tracking-javascript-guide.md

_paq.push(['trackPageView']);
```

If you want to ignore a special file type, you can just remove it from the list by using `removeDownloadExtensions( filetype )`:

``javascript

> // clicks on URLs finishing by png or mp4 will not be counted as downloads

Reviewer: really minor style improvement

```suggestion
// clicks on URLs ending in png or mp4 will not be counted as downloads
```
